app/cdap/components/WrangleHome/Components/OngoingDataExplorations/types.ts

```diff
@@ -78,3 +80,30 @@ export interface IMassagedObject {
   subText: string;
   workspaceId: string;
 }
+
+export interface IEachData {
+  connectionName: string;
+  count: number;
+  dataQuality: unknown;
+  recipeSteps: number;
+  workspaceId: string;
+  workspaceName: string;
+  connectorType: string;
+}
+
+export interface IMapValue {
```

Comment:
rename IMapValue to more appropriate one

Reply:
Done